deployment/default.yml.example

```diff
@@ -65,5 +65,7 @@ ITSIBaseURL: ''
 ITSISecretKey: ''
 HydroShareBaseURL: ''
 HydroShareSecretKey: ''
+SRATCatchmentAPIURL: ''
```

Comment:
Default values for these two need to be in `/var/lib/jenkins/.aws/staging.yml` on CivicCI for staging deployments to work again.

Reply:
Good catch. Added the values and triggered a deploy.

Reply:
Thanks!!